lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -193,7 +210,7 @@ if Enum.all?(
     end
 
     if Enum.all?(@opus_deps, &Code.ensure_loaded?/1) do
-      defp hls_links_and_children(link_builder, :OPUS, track_id, stream_id) do
+      defp hls_links_and_children(link_builder, :OPUS, track_id, stream_id, _segment_duration) do
```

Comment:
Mismatch arrity for second `hls_link_and_children` function for `:OPUS` encoding.